docs/sources/troubleshoot/support_bundle.md

```diff
@@ -0,0 +1,44 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/troubleshoot/support_bundle/
+description: Learn how to generate a support bundle
+title: Generate a Support Bundle
+menuTitle: Generate Support Bundle
+weight: 300
+---
+
+# Generate Support Bundle
```

Comment:
Will this be released as Public preview?

If yes, then we probably should add the Public preview bits at the top of the doc topic.

```
<span class="badge docs-labels__stage docs-labels__item">Public preview</span>

# Generate Support Bundle

{{< docs/shared lookup="stability/public_preview.md" source="alloy" version="<ALLOY_VERSION>" >}}
```

Reply:
Should we use the shared component for public_preview? The language doesn't seem quite right for this feature

<img width="765" alt="image" src="https://github.com/user-attachments/assets/28b525bc-1ec9-4a7c-a836-229596d69d55">

Reply:
Oooh good point. Lets switch to the generic one instead.  We could use this in its place: 

```
{{< docs/public-preview product="Generate support bundle" >
```

https://grafana.com/docs/writers-toolkit/write/shortcodes/#docspublic-preview